setup.py

```diff
@@ -251,16 +252,17 @@ def generate_cython():
         try:
             # Note, pip may not be installed or not have been used
             import pip
-            importlib.import_module('scipy/_lib/_pep440')
```

Comment:
Ugh, this was just never hit because the `subprocess` call higher up succeeds (almost) always. I didn't look at the change in the Pythran blog more closely because it was just a copy of this line.